axelor-sale/src/main/resources/domains/Cart.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<domain-models xmlns="http://axelor.com/xml/ns/domain-models"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://axelor.com/xml/ns/domain-models http://axelor.com/xml/ns/domain-models/domain-models_7.1.xsd">
+
+  <module name="sale" package="com.axelor.apps.sale.db"/>
+
+  <entity name="Cart">
+
+    <many-to-one name="user" column="user_id" title="User" ref="com.axelor.auth.db.User"
+      readonly="true"/>
```

Comment:
Remove the readonly from the domain and add it to the view. We will need to change the user in the future.